src/bootstrap/bootstrap.py

```diff
@@ -171,7 +171,7 @@ def download_stage0(self):
             if os.path.exists(self.bin_root()):
                 shutil.rmtree(self.bin_root())
             filename = "rust-std-{}-{}.tar.gz".format(channel, self.build)
-            url = "https://static.rust-lang.org/dist/" + self.stage0_rustc_date()
+            url = "https://dev-static.rust-lang.org/dist/" + self.stage0_rustc_date()
```

Comment:
It's unfortunate to make dev-static part of the public history, but for now fine, and I don't want to care too much.